src/@itwinjs-sandbox/view/ViewSetup.ts

```diff
@@ -89,7 +89,7 @@ export class ViewSetup {
         const modelIds = await ViewSetup.getModelIds(imodel);
         const subCategoryIds = await this.getSubCategoryIds(imodel, "S-SLAB-CONC");
         displayStyle.changeBackgroundMapProps({
-          planarClipMask: PlanarClipMaskSettings.create(PlanarClipMaskMode.IncludeSubCategories, modelIds, subCategoryIds),
+          planarClipMask: PlanarClipMaskSettings.createForElementsOrSubCategories(PlanarClipMaskMode.IncludeSubCategories, subCategoryIds, modelIds).toJSON(),
```

Comment:
Apparently you cannot leave a file level comment, or a comment on an unedited line... Unless I missed something.... But What is .ViewSetup.tsx.swp used for?

Reply:
Removed stray .swp file - this is used for vim which is my file editor.